Source/WebKit/WPEPlatform/wpe/WPEView.cpp

```diff
@@ -835,3 +835,23 @@ GList* wpe_view_get_preferred_dma_buf_formats(WPEView* view)
 
     return wpe_display_get_preferred_dma_buf_formats(view->priv->display.get());
 }
+
+/**
+ * wpe_view_set_opaque_rectangles:
+ * @view: a #WPEView
+ * @rects: (nullable) (array length=n_rects): opaque rectangles in view-local coordinates
```

Comment:
Why do we need a list of rects if we are only appending on rectangle in the code?

Reply:
Because the app might have more information when using a transparent background and can provide a different region.

Reply:
OK, then this is looking forward. For the future.